monitoring/managing-metrics.adoc

```diff
@@ -34,19 +34,27 @@ include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffs
 include::modules/monitoring-querying-metrics.adoc[leveloffset=+1]
 //include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+2]
 include::modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
```

Comment:
For readability and scanability, I think it reads better to have the link at the end of the sentence. Makes it easier for users to click the link, in my opinion. 

`For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation]`